torch/csrc/jit/script/init.cpp

```diff
@@ -481,6 +482,11 @@ void initJitScriptBindings(PyObject* module) {
         // There is a thin wrapper on top of this method in the C++ version of
         // ScriptModule.
         return runMethodFromPython(self.get_method("forward"), args);
+      })
```

Comment:
Adding this to the public API of Modules is confusing.  Instead, we should make this a stand alone function that takes a graph (not a Module) and returns a graph.

Reply:
@zdevito
@jamesr66a  said we may need to transform parameters in `ScriptModule` so I should transform based on `Module`?